doc/api-extensions.md

```diff
@@ -1709,3 +1709,8 @@ This introduces the storage bucket API. It allows the management of S3 object st
 
 ## `storage_buckets_create_credentials`
 This updates the storage bucket API to return initial admin credentials at bucket creation time.
+
+## `storage_buckets_local`
+This introduces the ability to use storage buckets on local storage pools.
+
+This introduces the `core.storage_buckets_address` global configuration setting.
```

Comment:
```suggestion

This introduces the ability to use storage buckets on local storage pools by setting the new `core.storage_buckets_address` global configuration setting.
```

Reply:
(Also adding a space after the heading to adhere to markdownlint in the future ;) )